### PR TITLE
initial nebula docs

### PR DESCRIPTION
**Pull request by saminacodes**
## Problem solved

Short description of the bug fixed or feature added NEB-16

---

## PR-Codex overview
This PR introduces the `Nebula` feature, enhancing the application with a new AI interface for blockchain interactions. It includes sidebar links, new icons, and documentation updates to support functionalities like gas fee payments and API interactions.

### Detailed summary
- Added `Nebula` link in `Header.tsx`.
- Introduced `ERC-20 Paymaster` link in `sidebar.tsx`.
- Created a new sidebar in `nebula/sidebar.tsx`.
- Implemented `Nebula` layout in `nebula/layout.tsx`.
- Updated `layout.tsx` with a new banner for `Nebula`.
- Added `NebulaIcon` in `icons/index.ts`.
- Developed `NebulaSection` in `page.tsx`.
- Created `NebulaIcon` component in `nebula/NebulaIcon.tsx`.
- Added content to `nebula/page.mdx` explaining its features.
- Documented `Nebula API Reference` in `api-reference/page.mdx`.
- Enhanced `ERC-20 Paymaster` documentation in `erc-20-paymaster/page.mdx`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/portal/src/app/page.tsx [hunk 1]
```diff
@@ -371,6 +365,23 @@ function InsightSection() {
   );
 }
 
+function NebulaSection() {
+  return (
+    <section className="my-12">
+      <SectionTitle id="insight" title="Insight" />
```
Comment (joaquim-verges) [inline apps/portal/src/app/page.tsx:371]: title still says insight

Comment (saminacodes) [reply apps/portal/src/app/page.tsx:371]: ah- thank you!

File: apps/portal/tsconfig.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
     "**/*.ts",
     "**/*.tsx",
     ".next/types/**/*.ts",
-    "src/app/react/v5/components/onchain/page.mdx"
+    "src/app/react/v5/components/onchain/page.mdx",
+    "src/app/nebula/page.mdx"
```
[the remark this answers is not in the document]
Comment (MananTank) [reply apps/portal/tsconfig.json:37]: No - its not needed. last 2 lines can be removed

[the remark this answers is not in the document]
Comment (MananTank) [reply apps/portal/tsconfig.json:37]: @saminacodes please remove: 

```
"src/app/react/v5/components/onchain/page.mdx",
"src/app/nebula/page.mdx"
```

**Pull request merged**